lib/authlete/model/request/token-revoke-request.rb

```diff
@@ -0,0 +1,57 @@
+# :nodoc:
+#
+# Copyright (C) 2014-2020 Authlete, Inc.
```

Comment:
I think this line should be "`Copyright (C) 2014-2024 Authlete, Inc.`"

Reply:
fixed: 7db0807a0c2f2669da6f331a8a50888b77cc3bb9